src/components/Countdown.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react'
+
+function Countdown({ Date }) {
+  const [date, setDate] = useState(Date)
```

Comment:
`Date` as parameter here looks like Date constructor.
Avoid using several-meaning in variable namings.